user/getting-started.md

```diff
@@ -89,6 +89,37 @@ $ echo "import foo" > sample.py
 $ bblfsh client sample.py
 ```
 
+### Overriding driver images
+
+Perhaps you need that Babelfish Server runs different driver images than the default ones. You can configure what images it should use throw the environment variable `BBLFSH_DRIVER_IMAGES`. The command `bblfsh server ` looks for this variable which must look like:
+
+    BBLFSH_DRIVER_IMAGES="language=docker-daemon:image:tag;language2=docker-daemon:image2:tag"
+
+So if you are running the server with a Docker container you could do something similar to:
+
+```
+$ docker run -e 'BBLFSH_DRIVER_IMAGES="python=docker-daemon:bblfsh/python:dev-96b24d3;java=docker-daemon:bblfsh/java-driver:latest"' --privileged -p 9432:9432 --name bblfsh bblfsh/server
+time="2017-07-12T14:11:13Z" level=debug msg="binding to 0.0.0.0:9432"
+time="2017-07-12T14:11:13Z" level=debug msg="initializing runtime at /tmp/bblfsh-runtime"
+time="2017-07-12T14:11:13Z" level=debug msg="Overriding image for "python: docker-daemon:bblfsh/python:dev-96b24d3"
+time="2017-07-12T14:11:13Z" level=debug msg="Overriding image for java: docker-daemon:bblfsh/java-driver:latest""
+time="2017-07-12T14:11:13Z" level=debug msg="starting server"
+time="2017-07-12T14:11:13Z" level=debug msg="registering gRPC service"
+time="2017-07-12T14:11:13Z" level=info msg="starting gRPC server"
+```
+Or if you prefer running in standalone mode:
```

Comment:
"running **it** in standalone mode" sounds better to me trough I think both expressions would be correct.